packages/sdk/README.md

```diff
@@ -1064,6 +1072,12 @@ client.loyalties.createMember(campaignId, member)
 client.loyalties.getMember(campaignId, memberId)
 ```
 
+#### [Get Loyalty Program Member Without CampaignId](https://docs.voucherify.io/reference/get-member-1)
```

Comment:
Few issues here:
1. The header and method name suggest that we do not need a campaign ID, but you expect it in params and link to documentation where the campaign ID is required.
2. I do not like naming. I understand that we have two similar endpoints, one that expects just member ID and the second one that needs campaign ID in addition. In documentation, both are described as `Get Member`. I guess (but we can confirm it) that the one that needs a campaign ID is a legacy one. We should give a name that will be intuitive for a developer who does not know about the context (legacy method). So, maybe `getMemberById`?